<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>defensecalculatorwindow</class>
 <widget class="QMainWindow" name="defensecalculatorwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>425</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="calculate_push_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>256</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>How Much
Damage Taken?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="armor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Armor</string>
    </property>
    <widget class="QLabel" name="armor_helmet_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Helmet</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="armor_helmet">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>69</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>69</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="armor_torso_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Torso</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="armor_torso">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="armor_gloves_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gloves</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="armor_gloves">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="armor_waist_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Waist</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="armor_waist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="armor_boots_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>217</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Boots</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="armor_boots">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="fortify">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fortify</string>
    </property>
    <widget class="QRadioButton" name="fortify_cart1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Cartx1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fortify_cart2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cartx&amp;2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fortify_none">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="total_defense_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total Defense:</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_defense_display">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="defense_up_kitchen">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Defense Up (Kitchen)</string>
    </property>
    <widget class="QRadioButton" name="defense_up_kitchen_small">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defense_up_kitchen_large">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Large</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defense_up_kitchen_none">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="defense_up_skill">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>61</y>
      <width>151</width>
      <height>129</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Defense Up (Skill)</string>
    </property>
    <widget class="QRadioButton" name="defense_up_skill_small">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defense_up_skill_medium">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Medium</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defense_up_skill_large">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Large</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defense_up_skill_extra_large">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>E&amp;xtra Large</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defense_up_skill_none">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hunting_horn">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>211</y>
      <width>151</width>
      <height>109</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Hunting Horn</string>
    </property>
    <widget class="QRadioButton" name="hunting_horn_small">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Defense (Small)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hunting_horn_large">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Defense (Large)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="hunting_horn_replay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Replay</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hunting_horn_none">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
